www/templates/main-goals.html

```diff
@@ -54,6 +55,7 @@
               <span ng-show="profile.items.currentPet" class="current-pet Pet-{{profile.items.currentPet}}"></span>
             </div>
           </figure>
+          <!--Habitica jade to html code ends here-->
```

Comment:
How does a `<div>` start have a `</figure>` end? Are you sure that the comments are in the correct place?